src/main/webapp/app/modules/provider/provider-record.reducer.ts

```diff
@@ -69,6 +77,11 @@ export default (state: ProviderRecordsState = initialState, action): ProviderRec
         updateSuccess: false,
         errorMessage: action.payload
       };
+    case FAILURE(ACTION_TYPES.CHECK_IN):
+      return {
+        ...state,
+        referralSent: false
```

Comment:
can we rename it to 'checkedIn' or 'beneficiaryCheckedIn'? The name will clash with sending actual referrals

Reply:
Fair point, will do.